Update convert-user-mailbox-to-shared-mailbox.md

dimcry opened the pull request:
Correcting the part of the article discussing about converting an user mailbox to a shared mailbox in an Exchange Hybrid environment.

dimcry committed:
Update convert-user-mailbox-to-shared-mailbox.md

Correcting the part of the article discussing about converting an user mailbox to a shared mailbox in an Exchange Hybrid environment.

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

dimcry committed:
Update convert-user-mailbox-to-shared-mailbox.md

JohanFreelancer9 commented on microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:
```diff
@@ -88,13 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
-Usually this is not a problem, but there are some scenarios where the on-premises attributes (which think that the mailbox is the user mailbox) can overwrite the new cloud versions of those attributes, and as a result, the mailbox might convert back. This is a problem because user mailboxes require licenses **or they are soft deleted after 30 days**!
-
-We've addressed most of the reasons why this happens but it still CAN happen, although infrequently. It's best to be safe and move the mailbox back to on-premises, convert it, and then move the shared mailbox back to the cloud. This recommended solution is not in violation of the licensing agreement for hybrid environments because the existence of the user mailbox on-premises is only temporary. Y ...
+More details about converting an user mailbox to a shared mailbox, into an Exchange Hybrid environment can be found on:
```
```suggestion
For details about converting a user mailbox to a shared mailbox in an Exchange hybrid environment, see [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49) and [Shared mailboxes are unexpectedly converted to user mailboxes after directory synchronization runs in an Exchange hybrid deployment](https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes).
```

dimcry replied:
Thank you!

JohanFreelancer9 commented on microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:
```diff
@@ -88,13 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
-
-We've addressed most of the reasons why this happens but it still CAN happen, although infrequently. It's best to be safe and move the mailbox back to on-premises, convert it, and then move the shared mailbox back to the cloud. This recommended solution is not in violation of the licensing agreement for hybrid environments because the existence of the user mailbox on-premises is only temporary. Y ...
+More details about converting an user mailbox to a shared mailbox, into an Exchange Hybrid environment can be found on:
+https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49
```
```suggestion
```

dimcry replied:
Thank you!

JohanFreelancer9 commented on microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:
```diff
@@ -88,13 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
-We've addressed most of the reasons why this happens but it still CAN happen, although infrequently. It's best to be safe and move the mailbox back to on-premises, convert it, and then move the shared mailbox back to the cloud. This recommended solution is not in violation of the licensing agreement for hybrid environments because the existence of the user mailbox on-premises is only temporary. Y ...
+More details about converting an user mailbox to a shared mailbox, into an Exchange Hybrid environment can be found on:
+https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49
+https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes
```
```suggestion
```

dimcry replied:
Thank you!

JohanFreelancer9 reviewed (changes requested):
Copy review.

dimcry commented:
@JohanFreelancer9 is something that I should continue / do, in order to have this change in Production?

JohanFreelancer9 commented:
@dimcry - yes, please merge the commit on the copy/edit review I've done. Thank you.

dimcry commented:
Merging the content in Production.

dimcry commented:
Seems that I don't have rights to commit...

dimcry commented:
@JohanFreelancer9 / @yogkumgit seems that I don't have rights to merge this in Production.
Can you please help me with this?

Thank you very much!

yogkumgit commented:
@dimcry and @JohanFreelancer9  confirm copy review

JohanFreelancer9 commented:
@yogkumgit - all good after my edits have been implemented.

dimcry commented:
@yogkumgit I confirm it can be published. Thank you!

yogkumgit commented:
@SKjerland Please merge this PR. Thanks.

SKjerland commented:
@yogkumgit or @JohanFreelancer9 - Please resolve the conflicts and then ping me again so I can merge. Thanks!

JohanFreelancer9 commented:
@yogkumgit - unfortunately I cannot resolve the conflicts, can you please assist?

yogkumgit commented:
@msbemba Please help and resolve this conflict if possible. Thanks

msbemba commented:
@yogkumgit I don't have access to resolve this conflict. Only those who created the PR can resolve the conflict.

SKjerland commented:
@dimcry - Can you please resolve the conflicts on this PR so that I can merge? Thanks!

dimcry committed:
Merge branch 'public' into patch-1

dimcry committed:
Update convert-user-mailbox-to-shared-mailbox.md

Removing the first Note and the Tip from the section, because the message was redundant with the message we've added into this section.
Please merge it in Prod.

dimcry committed:
Update convert-user-mailbox-to-shared-mailbox.md

Corrected "`Set-Mailbox -Identity mailbox1@contoso.onmicrosoft.com -Type Shared`" to "`Set-Mailbox -Identity mailbox1@contoso.com -Type Shared`"
In this example, the command is asked to be run in On-Premises. Customers should not use "<tenant>.onmicrosoft.com" domain in the On-Premises infrastructure, because this domains and changes related to this domain are not synced in Azure AD / MSODS. Is better to specify a vanity domain, instead of a Cloud default domain.

dimcry commented:
@yogkumgit / @msbemba  / @SKjerland / @JohanFreelancer9 please review and proceed in merging this change in Prod. Thank you!

yogkumgit commented:
@SKjerland
